src/templates/admin/core/accounts/edit_profile.html

```diff
@@ -0,0 +1,159 @@
+{% extends "admin/core/large_form.html" %}
+{% load i18n foundation static %}
+
+{% block css %}
+  <link type='text/css' href="{% static " common/css/jq-ui.css" %}" rel="stylesheet">
+{% endblock %}
+
+{% block contextual_title %}
+  {% trans "Edit Profile" %}
+{% endblock contextual_title %}
+
+{% block breadcrumbs %}
+  <li>Edit Profile</li>
+{% endblock breadcrumbs %}
+
+{% block title-section %}
+  {% trans "Edit Profile" %}
+{% endblock %}
+
+{% block body %}
+  <div class="grid">
+    {% include "admin/elements/forms/messages_in_callout.html" with form=form %}
+    <div class="max-w-56">
+      <section class="card padding-block-2 padding-inline-2">
+        <div class="title-area">
+          <h2>{% trans "Change Your Email Address" %}</h2>
+        </div>
+        {% blocktrans %}
+          <p>If you want to change your email address you may do so below,
+          however, you will be logged out and your
+          account will be marked as inactive until you follow the
+          instructions in the verification email. <strong>Note: </strong>
+          Changing your email address will also change your username as these
+          are one and the same.</p>
+        {% endblocktrans %}
+        <p><strong>{% trans 'Current Email Address' %}:</strong>
+        {{ request.user.email }}</p>
+        <form method="POST">
+          {% csrf_token %}
+          <div class="flex wrap column-gap-2">
+            <div>
+              <label for="email_address">
+                <strong>{% trans 'New Email Address' %}</strong>
+                <span aria-hidden="true">*</span>
+              </label>
+              <input type="email" name="email_address" required="true">
+            </div>
+          </div>
+          <button type="submit" name="email" class="button secondary">
+            {% trans "Update Email Address" %}
+          </button>
+        </form>
+        {% include "admin/elements/forms/denotes_required.html" %}
+      </section>
+      {% if request.journal and send_reader_notifications %}
+        <section class="card padding-block-2 padding-inline-2">
+          <div class="title-area">
+            <h2>{% trans "Register for Article Notifications" %}</h2>
+          </div>
+          <form method="POST">
+            <p>{% blocktrans %}
+              Use the button below to register to receive notifications of new articles
+              published in this journal.
+            {% endblocktrans %}</p>
+            {% csrf_token %}
+            {% if reader %}
```

Comment:
Spotted during testing that this is never set so the unsubscribe link is never displayed. Presumably its supposed to be in the context like:

```python
...
"reader": models.Role.objects.get(slug='reader'),
...
```

Reply:
OK I've fixed that.